### ISSUE TITLE
Temporal Duplication description wrong when locked

### ISSUE THREAD

**Issue by creidieki** [eternaldensity/Sandcastle-Builder#1473]
When Temporal Duplication is locked, its description reads "For ever, when you buy tools, get the same amount again for free!", even if you haven't earned permanent temporal duplication yet. (I tried to add it to my Favorites). Its description should probably check whether you have it (like Jamming, ASHF, etc.)

**Comment by eternaldensity** [comment]
Yeah a lot of boosts never expected to be visible when locked, so there's a lot of stuff which looks wrong when added to faves.

**Comment by Calamitizer** [comment]
At some point in the past, I went through all the boost I could think of that could re-locked after obtaining and accounted for that in the description (mostly just conditionally adding "When unlocked,"). I may have never pushed that update maybe? Or maybe I just missed TD.

**Comment by eternaldensity** [comment]
There's a bunch that conditionally add 'When active', so I figure you did push that.  TD would need a little more logic though since just adding 'when active' would still have the same wrong description listed here, so I can understand why you may have skipped it.

**Comment by barton2526** [comment]
Now it is impossible to see how many mNPs of this boost are remaining.

**Comment by eternaldensity** [comment]
This is what happens when I copypaste.  Thanks!

**Comment by eternaldensity** [comment]
Sorry, it's been two weeks an I still haven't gotten around to fixing my fix.  (Basically the condition is wrong because 'when active' for this boost is different to the other few I fixed at the same time.)
